### ISSUE TITLE
/etc/sysconfig/docker contains --live-restore 

### ISSUE THREAD

**Issue by visualex** [coreos/fedora-coreos-tracker#573]
The default installation of Docker has the live-restore feature enabled which is set in the file /etc/sysconfig/docker.

This does not seem to be the default with a normal docker install as mentioned here: 
[https://docs.docker.com/config/containers/live-restore/ ](https://docs.docker.com/config/containers/live-restore/ )
as well as not how the original CoreOS was before EOL. 

With this option set, it is not possible to join/start a swarm, therefore I need to remove it in the config yml file which is currently giving me problems.

1) What is the best way to override this setting, by adding my preferred /etc/sysconfig/docker file during installation?

2) Is this option a common requirement and should it stay in Fedora CoreOS?

thank you.

**Comment by lucab** [comment]
> What is the best way to override this setting, by adding my preferred /etc/sysconfig/docker file during installation?

Yes, or you can tweak the server unit and steer it towards your additional environment file. There is no optimal answer here, this kind of configuration via CLI flags has a lot of limits. See https://github.com/coreos/fedora-coreos-tracker/issues/588 too.

> Is this option a common requirement and should it stay in Fedora CoreOS?

This is the default from Fedora RPM, and we try not to tweak/diverge unless required. This setting is somehow workload specific, so I thinks it's better for users who need to change it to do it. We should highlight this different default in the migration doc though.
